etl/sql/stem/insert_drugs_into_stem.py

```diff
@@ -148,24 +142,29 @@ def get_drug_stem_insert(session: Any = None, logger: Any = None) -> Insert:
                 "__",
                 cast(Administrations.value, TEXT),
             ).label("source_value"),
-            OmopConcept.concept_id.label("source_concept_id"),
+            func.coalesce(OmopConcept.concept_id, ConceptLookupStem.uid).label(
+                "source_concept_id"
+            ),
             case(*quantity, else_=null()).label("quantity_or_value_as_number"),
             ConceptLookup.concept_id.label("route_concept_id"),
-            route_source_value,
+            func.coalesce(
```

Comment:
I'd lift this into a variable called something like `administration_route`, so it can be reused in multiple places.

Reply:
done